pkg/machine/qemu/machine.go

```diff
@@ -88,8 +88,9 @@ func NewMachine(opts machine.CreateOptions) (machine.VM, error) {
 
 	// Add network
 	cmd = append(cmd, "-nic", "user,model=virtio,hostfwd=tcp::"+strconv.Itoa(vm.Port)+"-:22")
+	// Disable graphic window
+	cmd = append(cmd, "-display", "none")
```

Comment:
can make this condition on not being in log-level=debug?

Reply:
Sure, I'll look into that